node/v2/handler.js

```diff
@@ -212,16 +212,25 @@ TChannelV2Handler.prototype.handleCallRequest = function handleCallRequest(reqFr
         )
     ));
 
-    channel.emitFastStat(channel.buildStat(
-        'connections.bytes-recvd',
-        'counter',
-        reqFrame.size,
-        new ConnectionsBytesRcvdTags(
-            channel.hostPort || '0.0.0.0:0',
-            self.connection.socketRemoteAddr
-        )
-    ));
+    self.emitBytesRecvd(reqFrame);
+};
 
+TChannelV2Handler.prototype.emitBytesRecvd =
+function emitBytesRecvd(frame) {
```

Comment:
Is there precedent for compressing "Received" to "Recvd" (I mean, if we ignore C libraries)?